core/controllers/home_test.py

```diff
@@ -26,8 +26,8 @@ class HomePageTest(test_utils.GenericTestBase):
     def test_logged_out_homepage(self):
         """Test the logged-out version of the home page."""
         response = self.testapp.get('/')
-        self.assertEqual(response.status_int, 200)
-        response.mustcontain(
+        self.assertEqual(response.status_int, 302)
```

Comment:
Asserted the response location through [this](https://github.com/526avijitgupta/oppia/commit/af9a9544081eb1500d912dea24a7ee786296af5d) commit